data_steward/tools/deid_runner.sh

```diff
@@ -135,14 +153,14 @@ data_stage='deid_clean'
 python "${CLEANER_DIR}/clean_cdr.py" --data_stage ${data_stage} -s 2>&1 | tee deid_clean_cleaning_log.txt
 
 # Create a snapshot dataset with the result
-python "${TOOLS_DIR}/snapshot_by_query.py" -p "${APP_ID}" -d "${cdr_deid_clean_staging}" -n "${cdr_deid_clean}"
+python "${TOOLS_DIR}/snapshot_by_query.py" --project_id "${APP_ID}" --dataset_id "${cdr_deid_clean_staging}" --snapshot_dataset_id "${cdr_deid_clean}"
 
-bq update --description "${version} De-identified Clean version of ${cdr_deid_base}" ${APP_ID}:${cdr_deid_clean}
+bq update --description "${version} De-identified Clean version of ${cdr_deid_base}" --label "type:deid_clean" --label "release_tag:${dataset_release_tag}" --label "de_identified:true" ${APP_ID}:${cdr_deid_clean}
```

Comment:
I referred to this while working on [DC-1069](https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=18&projectKey=DC&modal=detail&selectedIssue=DC-1069) and got this error:

`FATAL Flags parsing error: Unknown command line flag 'label'. Did you mean: table ?`

Figured out the update command doesn't have flag --label, I changed it to --set_label and it worked!

Reply:
@dcampbell-vumc Good catch!  Thank you!  I'll update my `bq update` commands.  :)